diff --git a/node_modules/expo-facebook/android/build.gradle b/node_modules/expo-facebook/android/build.gradle
index c8e7cb5..db0f3ca 100644
--- a/node_modules/expo-facebook/android/build.gradle
+++ b/node_modules/expo-facebook/android/build.gradle
@@ -1,4 +1,9 @@
 apply plugin: 'com.android.library'
+
+android {
+  compileSdk 36
+}
+
 apply plugin: 'kotlin-android'
 apply plugin: 'maven-publish'
 
@@ -37,7 +42,7 @@ buildscript {
 
 // Creating sources with comments
 task androidSourcesJar(type: Jar) {
-  classifier = 'sources'
+  archiveClassifier.set('sources')
   from android.sourceSets.main.java.srcDirs
 }
 
